packages/hs-react-ui/src/components/Dropdown/Dropdown.tsx

```diff
@@ -65,39 +73,70 @@ const ValueItem = styled(Div)`
 `;
 
 const OptionsContainer = styled(Div)`
-  background: white;
-  position: absolute;
-  top: 100%;
-  left: 0px;
-  max-height: 10rem;
-  overflow-y: scroll;
-  width: 15rem;
-  border: 0.5px solid ${colors.grayDark25};
-  border-radius: 0rem 0rem 0.25rem 0.25rem;
-  z-index: 1000;
+  ${({ multi, selected, variant, color }) => `
+    
+    /* use extra props for ts */
+    ${multi ? '' : ''}
+    ${variant ? '' : ''}
+    ${selected ? '' : ''}
```

Comment:
You should just be able to leave these out of the param declaration on line 76, right?

Reply:
Yes... and I can still add types for them even though I don't use them right?

Reply:
Right. The type declaration says that you need an object with attributes with those types. The param destructure doesn't care about the type declaration as far as typechecking goes.